### PR TITLE
Add fixes from user feedback

### PR DESCRIPTION
**Pull request by reillysiemens**
### Description

Fix some things based on user feedback.

### Check List

- [ ] All test passed
- [ ] Any new functionality or fixes you added is covered by new tests

All the tests fail. As designed. :wink:

### CODE REVIEW

File: calc/calc.py [hunk 1]
```diff
@@ -4,25 +4,29 @@
 produce a result.
 """
 
-from .token import Token, INTEGER, EOF, PLUS
+from calc.token import Token, INTEGER, EOF, PLUS, MINUS, TIMES, DIVIDED_BY
```
Comment (rawrgulmuffins) [inline calc/calc.py]: Was this intentionally added to master?

**Review by rawrgulmuffins — CHANGES_REQUESTED** [review]

File: calc/token.py [hunk 2]
```diff
@@ -6,9 +6,16 @@
 # Saved by the test: When these constants were defined the token strings were
 # inadvertently switched. The ``test_eof_token_at_end_of_line`` test caught the
 # error.
-INTEGER, EOF, PLUS = 'INTEGER', 'EOF', 'PLUS'
+INTEGER, EOF = 'INTEGER', 'EOF'
+PLUS, MINUS, TIMES, DIVIDED_BY = "PLUS", "MINUS", "TIMES", "DIVIDED_BY"
```
Comment (rawrgulmuffins) [inline calc/token.py]: Do we want to add these tokens to master

**Review by rawrgulmuffins — CHANGES_REQUESTED** [review]

File: calc/calc.py (hunk 1 above)
Comment (reillysiemens) [inline calc/calc.py]: I'll remove those.

File: calc/token.py (hunk 2 above)
Comment (reillysiemens) [inline calc/token.py]: I'll remove those.

**Review by rawrgulmuffins — APPROVED** [review]

**Pull request merged**